### PR TITLE
Sam

### PR DESCRIPTION
**Pull request by sussam**
Screen Shot 2021-04-08 at 1 24 40 PM

### CODE REVIEW

File: src/Styling/LandingPage.css [hunk 1]
```diff
@@ -10,8 +10,22 @@
...
+.landing-page p {
+    color: #F56F60;
+}
+
+.landing-welcome {
+    position: relative;
+    margin: 50px auto;
+}
+
+.landing-welcome img {
```
Comment (ngynkvn) [inline src/Styling/LandingPage.css:22]: Can these be tailwind classnames?

**Review by ngynkvn — COMMENTED** [review]
Looks good, just wondering about the `.landing-welcome` classes

**Review by rieckn — APPROVED** [review]

File: src/Styling/LandingPage.css (hunk 1 above)
Comment (sussam) [reply src/Styling/LandingPage.css]: I made the changes. Let me know if there's anything else you want me to change.

Comment (ngynkvn) [reply src/Styling/LandingPage.css]: Theres still merge confilcts on this branch, could you fix those?

Comment (sussam) [reply src/Styling/LandingPage.css]: Ahh ok I'm sorry. Fixed the merge conflicts!

**Pull request merged**